src/api_authentication_app_skeleton/src/actions/mixins/api/auth/helpers.cr

```diff
@@ -0,0 +1,21 @@
+module Api::Auth::Helpers(T)
```

Comment:
Maybe don't pass in `T` and instead hardcode `User`? Probably more clear. This is what people will use 99% of the time and if they don't want it they can change it